server/src/main/java/org/apache/cassandra/sidecar/db/schema/SidecarSchema.java

```diff
@@ -181,4 +186,23 @@ protected void reportSidecarSchemaInitialized()
     {
         vertx.eventBus().publish(ON_SIDECAR_SCHEMA_INITIALIZED.address(), "SidecarSchema initialized");
     }
+
+    /**
+     * Returns true when the schema should be created by this Sidecar instance. This currently
+     * depends on whether the election of single instance executor process is available, and
+     * the schemas are of type {@link InitializeOnSingleInstanceExecutor}, and the local Sidecar
+     * is a single instance executor. For all other types of schemas or if the election of leader
```

Comment:
Do we want to initialize the schema in every instance even if the we can't determine that we should be doing it? Could we defer the schema creation in that case, as we know (pre-TCM) this can cause significant issues with schema mismatches across instances?

Reply:
This is the status quo and not to be addressed in this PR. I think TCM will help here.